tracing-jaxrs-jakarta/src/main/java/com/palantir/tracing/jaxrs/JaxRsTracers.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tracing.jaxrs;
+
+import com.palantir.tracing.CloseableSpan;
+import com.palantir.tracing.Detached;
+import com.palantir.tracing.DetachedSpan;
+import com.palantir.tracing.Tracers;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.StreamingOutput;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.concurrent.Callable;
+
+public final class JaxRsTracers {
```

Comment:
I wonder if we could get away with silently dropping this feature/module in the jakarta migration. I see three uses across two projects internally, which could be handled in a couple lines of code.

Reply:
Ya, and I think I know one of them. I'm indifferent, it's not causing pain though.

Reply:
Ack, entirely up to you.